ios/Handlers/RNNativeViewHandler.m

```diff
@@ -99,21 +99,21 @@ - (void)handleTouchDown:(UIView *)sender forEvent:(UIEvent *)event
 
     [self sendEventsInState:RNGestureHandlerStateActive
              forViewWithTag:sender.reactTag
-              withExtraData:[RNGestureHandlerEventExtraData forPointerInside:YES]];
+              withExtraData:[RNGestureHandlerEventExtraData forPointerInside:YES withNumberOfTouches: _recognizer.numberOfTouches]];
```

Comment:
nit:
` withNumberOfTouches: _recognizer.numberOfTouches]];`

should be:
`withNumberOfTouches:_recognizer.numberOfTouches]];`

(no space after colon)

Same applies below